src/gluonts/mx/trainer/callback.py

```diff
@@ -338,3 +353,108 @@ def on_network_initializing_end(
         self, training_network: nn.HybridBlock
     ) -> None:
         copy_parameters(self.predictor.prediction_net, training_network)
+
+
+class TrainingTimeLimit(Callback):
+    def __init__(
+        self,
+        time_limit: int,
+        track_validation_duration: bool = True,
```

Comment:
I think something like `use_net_training` is better, since time can be spent in a lot of places, including validation.

Reply:
I'm not sure what this actually means.... What would we use ```use_net_training``` to record?